### PR TITLE
Extract plugin functions into utility methods and code style tweaks

### PR DESCRIPTION
**Pull request by kevva**
Just a continuation of https://github.com/wulkano/kap/pull/255 with some code style fixes.

### CODE REVIEW

File: app/src/main/plugins.js [hunk 1]
```diff
@@ -125,6 +125,28 @@ class Plugins {
...
+    }
+
+    return ret;
+  }
+
+  prettifyFormat(format) {
+    const ret = format.toUpperCase();
+    const formats = new Map([
+      ['WEBM', 'WebM']
+    ]);
```
Comment (sindresorhus) [inline app/src/main/plugins.js]: I know it's moot, but better for readability to just make a manual mapping of all of them, even if it works fine to just uppercase. Most consistent. Less differing code-paths.

File: app/src/main/touch-bar.js [hunk 2]
```diff
@@ -3,29 +3,21 @@ import plugins from './plugins';
...
-  const buttons = [
-    '16:9',
-    '5:4',
-    '5:3',
-    '4:3',
-    '3:2',
-    '1:1',
-    'Custom'
-  ].map(aspectRatio => {
+  const buttons = Array.from(aspectRatioToSize.keys()).map(aspectRatio => {
```
Comment (sindresorhus) [inline app/src/main/touch-bar.js]: Why not `[aspectRatioToSize.keys()...]`?

Comment (kevva) [reply app/src/main/touch-bar.js]: No reason. Forgot that it's supported now :).

**Comment by sindresorhus** [comment]
Looks good :)

**Pull request merged**